sklearn/datasets/descr/breast_cancer.rst

```diff
@@ -19,50 +19,50 @@ Data Set Characteristics:
         - concave points (number of concave portions of the contour)
         - symmetry 
         - fractal dimension ("coastline approximation" - 1)
-		
-		The mean, standard error, and "worst" or largest (mean of the three
-		largest values) of these features were computed for each image,
-		resulting in 30 features.  For instance, field 3 is Mean Radius, field
-		13 is Radius SE, field 23 is Worst Radius.
-		
+
+        The mean, standard error, and "worst" or largest (mean of the three
+        largest values) of these features were computed for each image,
+        resulting in 30 features.  For instance, field 3 is Mean Radius, field
+        13 is Radius SE, field 23 is Worst Radius.
+
         - class:
                 - WDBC-Malignant
                 - WDBC-Benign
 
     :Summary Statistics:
     ===================================== ====== ======
-										   Min    Max
+                                           Min    Max
     ===================================== ====== ====== 
     radius (mean):   					  6.981  28.11
     texture (mean):    					  9.71   39.28
     perimeter (mean):   				  43.79  188.5
     area (mean):    					  143.5  2501.0
-	smoothness (mean):					  0.053  0.163
-	compactness (mean):					  0.019  0.345
-	concavity (mean):					  0.0    0.427
-	concave points (mean):				  0.0	 0.201
-	symmetry (mean): 					  0.106  0.304
-	fractal dimension (mean):			  0.05	 0.097
+    smoothness (mean):					  0.053  0.163
+    compactness (mean):					  0.019  0.345
+    concavity (mean):					  0.0    0.427
+    concave points (mean):				  0.0	 0.201
+    symmetry (mean): 					  0.106  0.304
+    fractal dimension (mean):			  0.05	 0.097
```

Comment:
look at concave points in summary statistics. Also: building the website it the measure (i.e. how sphinx renders it).